package.json

```diff
@@ -11,7 +11,7 @@
   "types": "./index.d.ts",
   "type": "module",
   "engines": {
-    "node": "^12.20.0 || ^14.13.1 || >=16.0.0"
```

Comment:
Which dependency does this v16 requirement come from?

Reply:
that's one of the major changes with the new version of all unified/remark packages, eg.: https://github.com/unifiedjs/unified/commit/baf80b20d604c920a0656b4eec1c17612689f5ba